Tests/ComposableArchitectureTests/SharedTests.swift

```diff
@@ -461,7 +461,7 @@ final class SharedTests: XCTestCase {
     _count = Shared(0)
     let countDidChange = self.expectation(description: "countDidChange")
     withPerceptionTracking {
-      _ = $count
+      _ = $count.wrappedValue
```

Comment:
There's no need to observe a `Shared` when its underlying value is never accessed.